Application/EdFi.Security.DataAccess/Repositories/SecurityRepository.cs

```diff
@@ -5,119 +5,129 @@
 
 using System;
 using System.Collections.Generic;
-using System.Data.Entity;
 using System.Linq;
-using EdFi.Common;
-using EdFi.Common.Utils.Extensions;
-using EdFi.Security.DataAccess.Contexts;
 using EdFi.Security.DataAccess.Models;
+using Action = EdFi.Security.DataAccess.Models.Action;
 
 namespace EdFi.Security.DataAccess.Repositories
 {
-    public class SecurityRepository : SecurityRepositoryBase, ISecurityRepository
+    /// <summary>
+    /// Provides higher level access to raw security metadata.
+    /// </summary>
+    public class SecurityRepository : ISecurityRepository
     {
-        private readonly ISecurityContextFactory _securityContextFactory;
+        private readonly ISecurityTableGateway _securityTableGateway;
```

Comment:
Please revert this change, as it is likely to break the applications that are expecting to inject `ISecurityContextFactory`.